mri/parallel_mri/extract_sensitivity_maps.py

```diff
@@ -91,55 +92,39 @@ def extract_k_space_center_and_locations(data_values, samples_locations,
     return data_thresholded, center_locations
 
 
-def gridding_2d(points, values, img_shape, method='linear', point_min=None,
-                point_max=None):
+def grided_FT_ND(points, values, grid, method):
     """
-    Interpolate non-Cartesian data into a cartesian grid
+    This function calculates the grided fourier transform
+    from Interpolated non-Cartesian data into a cartesian grid
 
     Parameters
     ----------
     points: np.ndarray
-        The 2D k_space locations of size [M, 2]
+        The N-D k_space locations of size [M, N]
     values: np.ndarray
-        An image of size [N_x, N_y]
-    img_shape: tuple
-        The final output ndarray
+        An image of size [N_x, N_y, N_z, ..]
     method: {'linear', 'nearest', 'cubic'}, optional
         Method of interpolation for more details see scipy.interpolate.griddata
         documentation
-    points_min: float
-        The minimum points in the gridded matrix, if not provide take the min
-        of points
-    points_max: float
-        The maximum points in the gridded matrix, if not provide take the min
-        of points
+    grid: np.ndarray
+        The Gridded matrix for which you want to calculate k_space Smaps
     Returns
     -------
     np.ndarray
-        The gridded solution of shape [N_x, N_y]
+        The Sensitivity map for given channel
```

Comment:
It's not really a sensitivity map you are returning. It can be seen as a sensitivity map, but not necessarily. What you are returning is the inverse fourier transform of the grided (interpolated) values.

Reply:
I don't think this is resolved